DESCRIPTION

```diff
@@ -12,19 +12,19 @@ BugReports: https://github.com/reconhub/reportfactory/issues
 RoxygenNote: 6.1.1
 Imports:
     checkpoint,
+    earlyR,
```

Comment:
These should not go into imports. We don't use these packages directly for this package, so they should not be imported into the namespace. Keep the ones necessary only for the examples and tests in Suggests.

travis should be able to accommodate installing the packages in Suggests

Reply:
strange, I was getting a note or warning  that these were required when i ran the "check" function in Rstudio before, but when I reverted the file, it's not giving me the same output anymore. Will push up changes in a sec.

Reply:
> strange, I was getting a note or warning that these were required when i ran the "check" function in Rstudio before, but when I reverted the file, it's not giving me the same output anymore. Will push up changes in a sec.

It might be because you didn't have these packages installed before. I know I ran into that issue when I was working on this before.